src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -454,6 +454,147 @@ internal static string InsertSpacesToString(string original)
             return newText.ToString();
         }
 
+        /// <summary>
+        /// Gets words from text, e.g. ImportFromCSV to ("Import","From","CSV")
+        /// </summary>
+        /// <param name="text">incoming string</param>
+        /// <param name="maxCharacters">Max number of characters per row</param>
+        internal static IEnumerable<string> WrapText(string text, int maxCharacters)
```

Comment:
It seems like `maxCharacters` is not used here, that seems to be a mistake. From the `summary` above, it sounds like this method is the same as `InsertSpacesToString` method.